src/packages/core/src/sections/imageGallery1.tsx

```diff
@@ -1,15 +1,16 @@
 import React from 'react';
 
 import { getClassName } from '@kibalabs/core';
-import { Alignment, Direction, EqualGrid, PaddingSize, ResponsiveContainingView, ResponsiveField, ResponsiveTextAlignmentView, Stack, TextAlignment } from '@kibalabs/ui-react';
+import { Alignment, Direction, EqualGrid, MarkdownText, PaddingSize, ResponsiveContainingView, ResponsiveField, ResponsiveTextAlignmentView, Stack, TextAlignment } from '@kibalabs/ui-react';
 
 import { ISectionProps, Section } from '.';
 import { LazyMedia, SectionSubtitleText, SectionTitleText } from '../components';
 import { EverypagePaddingSize } from '../internal';
 
 interface IImageGallery1Image {
   mediaUrl: string;
-  title?: string;
+  altText?: string;
+  imgTitle?: string;
```

Comment:
dont use any shortenings please, we always use full words in kiba projects. pls make these `alternativeText` and `imageCaption`

Reply:
ah actually it would be better to keep title instead of caption, because otherwise its a breaking change

Reply:
in fact do we even need alternative text then?

Reply:
also, you need to add a prop to `IImageGallery1Props` below like `shouldShowImageCaptions` otherwise this is non-backwards compatible change (people will have captions even if they didnt  want to)

Reply:
ok